services/net/transcription/TranscriptionManager.cs

```
@@ -327,5 +337,50 @@ private async Task<string> RequestTranscriptionAsync(byte[] data, string languag
 
         return sb.ToString();
     }
+
+    /// <summary>
+    /// Check if the clip file contains a video stream.
+    /// </summary>
+    /// <param name="file"></param>
+    /// <returns></returns>
+    /// The same as Capture and Clip services, to be re-factored
+    private async Task<bool> IsVideoAsync(string file)
+    {
+        var process = new System.Diagnostics.Process();
+        process.StartInfo.Verb = $"Stream Type";
+        process.StartInfo.FileName = "/bin/sh";
+        process.StartInfo.Arguments = $"-c \"ffmpeg -i {file} 2>&1 | grep Video | awk '{{print $0}}' | tr -d ,\"";
+        process.StartInfo.UseShellExecute = false;
+        process.StartInfo.RedirectStandardOutput = true;
+        process.StartInfo.CreateNoWindow = true;
+        process.Start();
+
+        var output = await process.StandardOutput.ReadToEndAsync();
```

Any errors will be lost, we need to log them.